### ISSUE TITLE
scrollIntoView() or setScrollPosition() method?

### ISSUE THREAD

**Issue by Tibfib** [yads/ngTinyScrollbar#31]
I need to be able to update the scroll position. Is there some sort of `scrollIntoView()` method? So I could pass in a node and it would scroll until it was visible. Another, simpler, option would be a `setScrollPosition` method where I could find the element position myself and just update the scroller to where I need it to go. Is there something available to accomplish what I need currently? As of now I'm planning on just hacking the top position css... but that's not very pretty.

Thanks so much for ngTinyScrollbar – it's great!

**Comment by yads** [comment]
Yes, it exposes an [update](https://github.com/yads/ngTinyScrollbar/blob/master/src/ng-tiny-scrollbar.js#L100) method. Specify the position you want to scroll to in your outer directive.

**Comment by Mossar** [comment]
Can you help me how to use this update function?
For example I have: <div scrollbar></div> and how to call update(x) from the controller in which is this directive?

**Comment by yads** [comment]
@Mossar you would do it inside another directive's link function. So in your example your outer div would need to be another directive and when defining that controller you would add your update code in the outer directive's link function. See the documentation [here](https://docs.angularjs.org/guide/directive) under 'Creating Directives that Communicate'
